common/utils.js

```diff
@@ -785,6 +785,40 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function estimateITSFee(chain, destinationChain, env, eventType, gasValue, _axelar) {
+    if (env === 'devnet-amplifier') {
+        printInfo('Using default ITS fee: 0');
```

Comment:
Verbose logging in helper method isn't needed (user doesn't really care about knowing this as long as it works)